### ISSUE TITLE
Eject disk and lock keyboard feature 

### ISSUE THREAD

**Issue by allen5218** [jacklandrin/OnlySwitch#56]
I switched from one switch to only switch, and only switch is better than ones witch in most aspects, but it lacks the eject disk and locking keyboard function. These two features are very useful to me, and I hope they can be added in the future.

**Comment by jacklandrin** [comment]
Sure. I will consider adding this feature.

**Comment by dylandea** [comment]
I think "eject disk" was added right?
And yeah I'd also love possibility to lock keyboard, for cleaning it or for my child using the computer lol

**Comment by jacklandrin** [comment]
@dylandea Yes, 2.3.9 has supported eject disk. About cleaning keyboard, you can use Screen Test temporarily.

**Comment by dylandea** [comment]
Thanks I'll try when I get home.
I am currently using an app by the creator of BetterTouchTool for the keyboard.

But if I could keep only one app for all this (Only Switch), that'd be great.

**Comment by jacklandrin** [comment]
@dylandea No problem. Screen Test is just a workaround. If you press some special shortcuts, they also will response, such as cmd+Q and the power button.

**Comment by jacklandrin** [comment]
@dylandea I tried to lock special keys and shortcuts such as the power button and cmd+Q in Screen Test. This is the beta package.
[OnlySwitch_beta.zip](https://github.com/jacklandrin/OnlySwitch/files/10261832/OnlySwitch_beta.zip)
